### PR TITLE
Fix compilation issue in WinSim Demo

### PR DESCRIPTION
**Pull request by kar-rahul-aws**
Description
-----------
This PR fixes the compilation issue in the FreeRTOS+TCP WinSim Demo. 

Test Steps
-----------
Compile and run the [Demo](https://github.com/FreeRTOS/FreeRTOS/tree/main/FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_Minimal_Windows_Simulator)

Checklist:
----------

- [x] I have tested my changes. No regression in existing tests.
- [x] I have modified and/or added unit-tests to cover the code changes in this Pull Request.

Related Issue
-----------

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_Minimal_Windows_Simulator/main.c [hunk 1]
```diff
@@ -185,14 +186,15 @@ int main( void )
         }
         #endif /* ( ipconfigUSE_DHCP != 0 ) */
 
-        memcpy( ipLOCAL_MAC_ADDRESS, ucMACAddress, sizeof( ucMACAddress ) );
-
-        FreeRTOS_IPInit_Multi();
+        /*memcpy( ipLOCAL_MAC_ADDRESS, ucMACAddress, sizeof( ucMACAddress ) ); */
```
Comment (moninom1) [inline FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_Minimal_Windows_Simulator/main.c:189]: Can this be removed?

Comment (ActoryOu) [inline FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_Minimal_Windows_Simulator/main.c:189]: Can we just remove this line instead of comment out?

Comment (kar-rahul-aws) [reply FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_Minimal_Windows_Simulator/main.c]: updated.

Comment (kar-rahul-aws) [reply FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_Minimal_Windows_Simulator/main.c]: updated.

**Review by tony-josi-aws — APPROVED** [review]

**Review by moninom1 — APPROVED** [review]

**Review by shubnil — APPROVED** [review]

**Pull request merged**